### PR TITLE
Add secret rotation to weekly build

### PR DESCRIPTION
**Pull request by alexperovich**

### CODE REVIEW

File: eng/synchronize-secrets.ps1 [hunk 1]
```diff
@@ -0,0 +1,10 @@
+param(
```
Comment (ChadNedzlek) [inline eng/synchronize-secrets.ps1:1]: This entire script is basically just a foreach... is the script really necessary? The call could just be:
`dir .vault-config/*.yaml |% { dotnet run -p }`
The script indirection really isn't adding much.

**Review by ChadNedzlek — CHANGES_REQUESTED** [review]

File: eng/synchronize-secrets.ps1 (hunk 1 above)
Comment (ChadNedzlek) [reply eng/synchronize-secrets.ps1:1]: And then you don't hvae to deal with the weird string escaping, or the always scary "Invoke-Expression"

Comment (alexperovich) [reply eng/synchronize-secrets.ps1:1]: yea, thats true. I'll do that.

Comment (alexperovich) [reply eng/synchronize-secrets.ps1:1]: When I first wrote It I thought the script would have more in it.

**Review by ChadNedzlek — APPROVED** [review]

**Review by jakubstilec — APPROVED** [review]

**Review by MattGal — APPROVED** [review]

**Pull request merged**